request/request.go

```diff
@@ -52,7 +52,7 @@ func (r *Request) WithTables(tables ...*table.Table) *Request {
 }
 
 func (r *Request) IsZero() bool {
-	return r.tables == nil || len(r.tables) == 0
+	return r.tables == nil
```

Comment:
suggestion: is it better to use `IsNilTable()` or `IsTableSet()`?

Reply:
I modified it.